src/node_crypto.cc

```diff
@@ -792,6 +799,9 @@ static X509_STORE* NewRootCertStore() {
   }
 
   X509_STORE* store = X509_STORE_new();
+  if (strlen(system_cert_path) != 0) {
```

Comment:
Not wrong but I'd write this as `if (*system_cert_path != '\0') {`.

Reply:
I'll change this. Thanks